server/services/buoy/seed_buoys.py

```diff
@@ -39,7 +39,7 @@ def parse_activestations(stations):
         for station in stations:
             station = station.attrs
             lon, lat = float(station.get("lon", 0.0)), float(station.get("lat", 0.0))
-            geo = f"Point({lon} {lat})"
+            geo = f"POINT({lon} {lat})"
```

Comment:
Lol why? This was unnecessary. Was it OCD?

Reply:
I cannot not conform nor deny... 👀